### PR TITLE
Update index.md - add link to resource in project, article, video and community section

### PR DESCRIPTION
**Pull request by taminobaumann**
## Description

Links to learning resources (White Papers), a community, a project (Opens-Source, working across ecosystems) and an explainer/demo video to help developers get started with building.

### CODE REVIEW

File: src/content/decentralized-identity/index.md [hunk 1]
```diff
@@ -170,16 +171,19 @@ There are many ambitious projects using Ethereum as a foundation for decentraliz
...
 - [BrightID - Decentralized Identity on Ethereum](https://www.youtube.com/watch?v=D3DbMFYGRoM) — _Bankless podcast episode discussing BrightID, a decentralized identity solution for Ethereum_
 - [The Off Chain Internet: Decentralized Identity & Verifiable Credentials](https://www.youtube.com/watch?v=EZ_Bb6j87mg) — EthDenver 2022 presentation by Evin McMullen
+- [Verifiable Credentials Explained](https://www.youtube.com/watch?v=ce1IdSr-Kig) - YouTube explainer video with demo by Tamino Baumann
 
 ### Communities {#communities}
 
 - [ERC-725 Alliance on GitHub](https://github.com/erc725alliance) — _Supporters of the ERC725 standard for managing identity on the Ethereum blockchain_
 - [SpruceID Discord server](https://discord.com/invite/Sf9tSFzrnt) — _Community for enthusiasts and developers working on Sign-in with Ethereum_
 - [Veramo Labs](https://discord.gg/sYBUXpACh4) — _A community of developers contributing to building a framework for verifiable data for applications_
+- [walt.id](https://discord.com/invite/AW8AgqJthZ) — _A community of developers and builders working on decentralized identity use cases across various industries_
```
Comment (minimalsm) [inline src/content/decentralized-identity/index.md:189]: Not a strong take, but not sure how relevant this particular community is to Ethereum specifically

File: src/content/decentralized-identity/index.md [hunk 2]
```diff
@@ -170,16 +171,19 @@ There are many ambitious projects using Ethereum as a foundation for decentraliz
 - [What is Ethereum ERC725? Self-Sovereign Identity Management on the Blockchain](https://cryptoslate.com/what-is-erc725-self-sovereign-identity-management-on-the-blockchain/) — _Sam Town_
 - [How Blockchain Could Solve the Problem of Digital Identity](https://time.com/6142810/proof-of-humanity/) — _Andrew R. Chow_
 - [What Is Decentralized Identity And Why Should You Care?](https://web3.hashnode.com/what-is-decentralized-identity) — _Emmanuel Awosika_
+- [Introduction to Decentralized Identity](https://walt.id/white-papers)  — _Dominik Beron_
```
Comment (minimalsm) [inline src/content/decentralized-identity/index.md:174]: Is there a starting point for users? Landing them on a page with 7 complex articles doesn't seem ideal. Personally, I'd prefer to link directly to the most beginner friendly.

File: src/content/decentralized-identity/index.md [hunk 3]
```diff
@@ -161,6 +161,7 @@ There are many ambitious projects using Ethereum as a foundation for decentraliz
 - **[Proof of Humanity](https://www.proofofhumanity.id)** - _Proof of Humanity (or PoH) is a social identity verification system built on Ethereum._
 - **[BrightID](https://www.brightid.org/)** - _A decentralized, open-source social identity network seeking to reform identity verification through the creation and analysis of a social graph._
 - **[Proof-of-personhood Passport](https://proofofpersonhood.com/)** - _A decentralized digital identity aggregator._
+- **[walt.id](https://walt.id)** — _Open source decentralized identity and wallet infrastructure that enables developers and organizations to leverage self-sovereign identity and NFTs/SBTs._
```
[the remark this answers is not in the document]
Comment (taminobaumann) [reply src/content/decentralized-identity/index.md:164]: Agree, it is more for developers than users.

File: src/content/decentralized-identity/index.md (hunk 1 above)
Comment (taminobaumann) [reply src/content/decentralized-identity/index.md:189]: With our upcoming update, our solution will also hold support for did:ethr, which I think holds value for the community.

File: src/content/decentralized-identity/index.md (hunk 2 above)
Comment (taminobaumann) [reply src/content/decentralized-identity/index.md]: Good point, thanks!! I've updated it to the most beginner-friendly one.

**Comment by taminobaumann** [comment]
@minimalsm thanks a lot for the review and the great feedback!

File: src/content/decentralized-identity/index.md (hunk 1 above)
Comment (wackerow) [reply src/content/decentralized-identity/index.md:189]: I see both points... I think this sounds reasonable though given this somewhat chain-agnostic topic.

**Review by wackerow — APPROVED** [review]
lgtm, thanks @taminobaumann! going to bring this in

**Comment by wackerow** [comment]
@all-contributors please add @taminobaumann for content

**Pull request merged**